experimental/oak_baremetal_loader/src/main.rs

```diff
@@ -58,8 +59,17 @@ struct Args {
     mode: Mode,
 
     /// Path to the kernel to execute.
-    #[clap(parse(from_os_str), validator = path_exists)]
+    #[clap(long, parse(from_os_str), validator = path_exists)]
     app: PathBuf,
+
+    /// Path to a Wasm file to be loaded into he trusted runtime and executed by it per invocation.
```

Comment:
typo: he -> the trusted runtime. I think it must also export `alloc`, but maybe not? Does it help to link to [ABI](https://github.com/project-oak/oak/blob/main/docs/oak_functions_abi.md)?